### PR TITLE
Update, correct, and add badges/links

### PR DESCRIPTION
**Pull request by oscargus**
## PR Summary
* Change to shields.io badges for some.
* Add conda version.
* Direct links to pypi.
* Link to devel-section of documentation instead of git book. (Maybe should be two links?)
* Better codecod-link.

## PR Checklist

**Linked Issue**
- [ ] Added "closes #0000" in the PR description to link it to the original issue.

**Documentation and Tests**
- [ ] Has pytest style unit tests (and `pytest` passes)
- [ ] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).
- [ ] New plotting related features are documented with examples.

**Release Notes**
- [ ] New features are marked with a `.. versionadded::` directive in the docstring and documented in `doc/users/next_whats_new/`
- [ ] API changes are marked with a `.. versionchanged::` directive in the docstring and documented in `doc/api/next_api_changes/`
- [ ] Release notes conform with instructions in  `next_whats_new/README.rst` or `next_api_changes/README.rst`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
+[![PyPi](https://img.shields.io/pypi/v/matplotlib)](https://pypi.org/project/matplotlib/)
+[![Conda](https://img.shields.io/conda/vn/conda-forge/matplotlib)](https://anaconda.org/conda-forge/matplotlib)
+[![Downloads](https://img.shields.io/pypi/dm/matplotlib)](https://pypi.org/project/matplotlib)
 [![NUMFocus](https://img.shields.io/badge/powered%20by-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A)](https://numfocus.org)
 
 [![DiscourseBadge](https://img.shields.io/badge/help_forum-discourse-blue.svg)](https://discourse.matplotlib.org)
 [![Gitter](https://badges.gitter.im/matplotlib/matplotlib.svg)](https://gitter.im/matplotlib/matplotlib)
 [![GitHubIssues](https://img.shields.io/badge/issue_tracking-github-blue.svg)](https://github.com/matplotlib/matplotlib/issues)
-[![GitTutorial](https://img.shields.io/badge/PR-Welcome-%23FF8300.svg?)](https://git-scm.com/book/en/v2/GitHub-Contributing-to-a-Project)
+[![GitTutorial](https://img.shields.io/badge/PR-Welcome-%23FF8300.svg?)](https://matplotlib.org/stable/devel/index.html)
```
Comment (QuLogic) [inline README.md:9]: This doesn't seem to be a Git tutorial.

Comment (oscargus) [reply README.md:9]: Yes, you are correct. Where does this text actually show up? I'm not saying that it shouldn't be correct though.

Should we also add a Git Tutorial bagde? Like https://img.shields.io/badge/git-tutorial-brightgreen

Comment (timhoffm) [inline README.md:9]: Change the alt-text
```suggestion
[![Contribute](https://img.shields.io/badge/PR-Welcome-%23FF8300.svg?)](https://matplotlib.org/stable/devel/index.html)
```

Comment (timhoffm) [reply README.md:9]: No. IMHO that's too specific. Git explanations are/should be part of the contribution guide. This link to the contribution guide is enough here.

Comment (QuLogic) [reply README.md:9]: > Yes, you are correct. Where does this text actually show up? I'm not saying that it shouldn't be correct though.

It appears in the image's alt text.

Comment (oscargus) [reply README.md:9]: That is what I thought, but as part of that I have now realized that my browser doesn't show alt-texts at all...

**Comment by oscargus** [comment]
As it is the alt-text for the images, I also updated some of those to make (a bit) more sense for those why may use them...

**Review by timhoffm — APPROVED** [review]

**Pull request merged**